src/rtext.c

```diff
@@ -1402,23 +1411,13 @@ int TextToInteger(const char *text)
 // Copy one string to another, returns bytes copied
 int TextCopy(char *dst, const char *src)
 {
-    int bytes = 0;
-
-    if ((src != NULL) && (dst != NULL))
-    {
-        while (*src != '\0')
-        {
-            *dst = *src;
-            dst++;
-            src++;
-
-            bytes++;
-        }
-
-        *dst = '\0';
-    }
+    if (!dst || !src)
+        return 0;
 
-    return bytes;
+    // use strcpy since it uses vector operations
+    unsigned int length = strcpy(dst, src);
```

Comment:
Just as a reference, this causes an `initialization of ‘unsigned int’ from ‘char *’ makes integer from pointer without a cast`. It can be fixed with `unsigned int length = *strcpy(dst, src);`.

After the fix, tested this `TextCopy()` and it showed a performance improvement for `PLATFORM_DESKTOP`:
```
~/test$ ./test
TextCopy:  0.000007 sec (abcdefghijklmnopqrstuvxwyz)
TextCopy2: 0.000002 sec (abcdefghijklmnopqrstuvxwyz)
~/test$ ./test
TextCopy:  0.000007 sec (abcdefghijklmnopqrstuvxwyz)
TextCopy2: 0.000001 sec (abcdefghijklmnopqrstuvxwyz)
~/test$ ./test
TextCopy:  0.000007 sec (abcdefghijklmnopqrstuvxwyz)
TextCopy2: 0.000001 sec (abcdefghijklmnopqrstuvxwyz)
```
```
#include <stdio.h>
#include <time.h>
#include "raylib.h"

int main(void) {

    {   char testRes[30] = "0";
        double timeRes = 0.0f;
        clock_t timeStart = clock();
        TextCopy(testRes, "abcdefghijklmnopqrstuvxwyz");
        clock_t timeEnd = clock();
        timeRes = (double)(timeEnd - timeStart) / CLOCKS_PER_SEC;
        printf("TextCopy:  %f sec (%s)\n", timeRes, testRes); }

    {   char testRes[30] = "";
        double timeRes = 0.0f;
        clock_t timeStart = clock();
        TextCopy2(testRes, "abcdefghijklmnopqrstuvxwyz");
        clock_t timeEnd = clock();
        timeRes = (double)(timeEnd - timeStart) / CLOCKS_PER_SEC;
        printf("TextCopy2: %f sec (%s)\n", timeRes, testRes); }

    return 0;
}
```